### DISCUSSION TITLE
Getting started... sort of

### DISCUSSION THREAD

**Discussion by noconsulate** [supabase/supabase#1106] [Feature Requests] [answered]
The devs appear to be motivated to get people using Supabase by being active in forums and answering questions, almost to the point of hand-holding! For me, as a relatively novice developer, this is great and having the opportunity to get questions answered is always a big plus when approaching a new tool or tech. 

The documentation in its current form is good enough to "get started" but it's hard to find the relevant instructions. In fact I had started writing this post to complain that I couldn't find any actual docs for the client and decided to try one more time and finally found [this](https://supabase.io/docs/reference/javascript/supabase-client). I'm glad I had the link in my clipboard because I just went back to click around supabase.io a bit more and I'm not exactly sure where I found it! 

I'm not a designer or ux-pert so it's hard for me to pin down what's wrong but it just seems like the links are a mess and important stuff is hidden, especially in regards to the client. Most of the attention seems to be on the database and dashboard, which is understandable, but the client is equally important because one can't do much without it. 

I might suggest a more traditional 'getting started' doc wherein you walk the user through starting a database, initializing the client, and doing some CRUD. The example projects you offer are excellent but can be cumbersome to learn from because even if I know the exact stack of one of the examples I still have to process foreign code. With a very simple walk-through from scratch I know each line of code is for and I don't have to work out the logic of a strange app just to get the goods on how to do operations with the client.

One more thing...! Clearly you guys wants to attract Firebase users, which is great. The thing about Firebase users (I'm speaking for myself here) is that they often don't know much about databases and especially SQL. If they did they might not be using Firebase. Supabase makes dealing with postgres easy but it doesn't hide away the intricacies and it seems impossible to use it meaningfully without some knowledge of SQL. To me, a lowly Firebase user, data is little more than an object I talk to with Javascript. So my suggestion is you link to an excellent and concise primer to SQL queries and tables, and eventually maybe develop your own. Aspects of postgresql that can't be abstracted away should be taught to the user because you are. after all, dealing with Firebase users!

Anyway, great work. I'm going to keep playing around with Supabase for a few more days before I decide to use it or stay with Parse.

**Comment by silentworks** [comment, marked as answer]
This is really good feedback. I think the documentation hasn't been updated for a while now and will probably get some looking at in the future (I don't work at supabase so not sure of dates). I like the idea that getting started should be a walkthrough of getting the entire thing set up and also the idea of having a SQL primer for those who are not familiar with SQL.

**Reply by kiwicopple** [reply]
What @silentworks has mentioned is accurate - we're very aware that our docs aren't catering for a large portion of the developers. They need improvement across the board - guides, reference docs, tutorials, and examples.

Now that we've finished Launch Week we have more time to focus on this. You've given some great feedback @noconsulate - we'll revisit this comment often as we start updating. Thanks!
